packages/lib/src/types/dataPasser.ts

```diff
@@ -30,4 +30,5 @@ export interface LensDataPasser extends HTMLElement {
     removeValueFromQueryAPI(params: RemoveValueFromQueryAPIParams): void;
     updateResponseAPI(params: ResponseStore): void;
     setQueryStoreAPI(params: QueryItem[][]): void;
+    setQueryStoreFromAstAPI(params: QueryItem[][]): void;
```

Comment:
This function name does not match the one in the definition. Literally can't test 😉 The Rest of the code looks good.

Reply:
Okay, i will adjust the internal name in the datapasser web component to match the one mentioned here.